xml/glx.xml

```diff
@@ -1051,11 +1051,11 @@ typedef unsigned __int64 uint64_t;
         </command>
         <command>
             <proto><ptype>__GLXextFuncPtr</ptype> <name>glXGetProcAddressARB</name></proto>
-            <param>const <ptype>GLubyte</ptype> *<name>procName</name></param>
+            <param kind="String">const <ptype>GLubyte</ptype> *<name>procName</name></param>
         </command>
         <command>
             <proto><ptype>__GLXextFuncPtr</ptype> <name>glXGetProcAddress</name></proto>
-            <param>const <ptype>GLubyte</ptype> *<name>procName</name></param>
+            <param kind="String">const <ptype>GLubyte</ptype> *<name>procName</name></param>
```

Comment:
I don't fully remember, but didn't we agree on something like `gl::String` to reference another XML document?

Reply:
I was under the impression that that was for enumerated groups only. I think we could possibly extend it but then I think that requires further changes to the xml schema

Reply:
Ah, right, this is the first time the `kind` attribute appeared outside of `gl.xml`.

Well, it's only slightly bugging me that the description is in another XML file.

I'm fine with keeping kinds non-namespaced, in case you think we wouldn't need that separation, but let's agree on this now.

Reply:
Personally I'm fine with both.
I personally think we won't need namespaces for kinds as I don't think we'll need the same name to mean different things in the different xml files.